kubernetes/configs/dev-refactor/commons.sh

```diff
@@ -15,6 +15,7 @@ READ_MODEL_REPLICA_SET="rs0"
 READ_MODEL_READ_PREFERENCE="secondaryPreferred"
 REPLICAS=1
 BACKEND_SERVICE_PORT="8088"
+KAFKA_BROKERS="boot-yqksbq44.c3.kafka-serverless.eu-central-1.amazonaws.com:9098"
```

Comment:
we could use this also in `catalog-readmodel-writer` configmap

Reply:
Yeah, I thought the same while I was writing but I wanted to keep this PR specific to `authorization-updater`. I'll open another PR after this one